pycti/entities/stix_cyber_observable/opencti_stix_cyber_observable_deprecated.py

```diff
@@ -0,0 +1,56 @@
+import deprecation
+
+
+class StixCyberObservableDeprecatedMixin:
+    """
+    deprecated >=6.2 & <6.5. Use `promote_to_indicator_v2`
+    Promote a Stix-Observable to an Indicator
+
+    :param id: the Stix-Observable id
+    :return the observable
+    """
+
+    @deprecation.deprecated(
+        deprecated_in="6.2",
+        removed_in="6.5",
+        details="Use promote_to_indicator_v2 instead.",
+    )
```

Comment:
This will print a warning message if the method is used, allowing our users to maintain their scripts more easily, hopefully.